js/interface.js

```diff
@@ -2382,14 +2382,10 @@ function getSecurityRule() {
   }
 
   if (currentFinalRules.length > 0) {
-    currentFinalRules.forEach(function(rule) {
-      if (widgetData.tokenId) {
-        if (rule.allow.hasOwnProperty('tokens') && Number(rule.allow.tokens[0]) === widgetData.tokenId) {
-          hasSecurityRule = true;
-        }
-      } else if (rule.allow.hasOwnProperty('tokens') && Number(rule.allow.tokens[0]) === selectedTokenId) {
-        hasSecurityRule = true;
-      }
+    hasSecurityRule = currentFinalRules.some(function(rule) {
+      return _.some(rule.allow.tokens, function(token) {
+        return token && (token === widgetData.tokenId || token === selectedTokenId);
```

Comment:
Is `Number()` needed or not? You've kept it in your previous iteration of the code, but not this time.

Reply:
Yes now value is coming as Number so not added.